functional-tests/src/main/java/com/redhat/che/selenium/core/workspace/CheStarterWrapper.java

```diff
@@ -0,0 +1,224 @@
+    requestBuilder.addHeader("Content-Type", "application/json");
+    requestBuilder.addHeader("Authorization", "Bearer " + token);
+    Request request = requestBuilder.post(body).build();
+    OkHttpClient client = new OkHttpClient();
+    try {
+      Response response = client.newCall(request).execute();
+      return getNameFromResponse(response);
+    } catch (IOException e) {
+      LOG.error("Workspace could not be created : "+e.getMessage(), e);
+      return null;
+    }
+  }
+
+  public boolean deleteWorkspace(String workspaceName, String token) {
+    StringBuilder sb = new StringBuilder(this.cheStarterURL);
+    String path = "/workspace/" + workspaceName;
+    sb.append(path);
+    sb.append("?");
+    sb.append("masterUrl=").append(this.host).append("&").append("namespace=sth");
+    Builder requestBuilder = new Request.Builder().url(sb.toString());
+    requestBuilder.addHeader("Content-Type", "application/json");
+    requestBuilder.addHeader("Authorization", "Bearer " + token);
+    OkHttpClient client = new OkHttpClient();
+    try {
+      Response response = client.newCall(requestBuilder.delete().build()).execute();
+      LOG.info("Workspace delete response : "+response.message());
+      return response.isSuccessful();
+    } catch (IOException e) {
+      LOG.error("Workspace could not be deleted : "+e.getMessage(), e);
+      return false;
+    }
+  }
+
+  public void startWorkspace(String WorkspaceID, String name, String token) throws Exception {
+    OkHttpClient client = new OkHttpClient();
+    String path = "/workspace/" + name;
+    StringBuilder sb = new StringBuilder(this.cheStarterURL);
+    sb.append(path);
+    sb.append("?");
+    sb.append("masterUrl=").append(this.host).append("&").append("namespace=sthf");
+    Builder requestBuilder = new Request.Builder().url(sb.toString());
+    requestBuilder.addHeader("Authorization", "Bearer " + token);
+    RequestBody body = RequestBody.create(null, new byte[0]);
+    Request request = requestBuilder.patch(body).build();
+    try {
+      Response response = client.newCall(request).execute();
+      if (response.isSuccessful()) {
+        LOG.info("Prepare workspace request send. Starting workspace.");
+        sb = new StringBuilder("https://" + this.host);
+        sb.append("/api/workspace/");
+        sb.append(WorkspaceID);
+        sb.append("/runtime");
+        requestBuilder = new Request.Builder().url(sb.toString());
+        requestBuilder.addHeader("Authorization", "Bearer " + token);
+        request = requestBuilder.post(body).build();
+        response = client.newCall(request).execute();
+        if (response.isSuccessful()) {
+          LOG.info("Workspace was started. Waiting until workspace is running.");
+        }
+      }
+    } catch (IOException e) {
+      LOG.error("Workspace start failed : "+e.getMessage(), e);
+      throw e;
+    }
+  }
+
+  // ================= //
+  //  PRIVATE METHODS  //
+  // ================= //
```

Comment:
Do not think that this `PRIVATE METHODS` block is relevant. Private methods should always be in the end of the class, so no need to specify it explicitly via comments

Reply:
It catches the eye easily, but I'm willing to remove it if requested.